gui/src/app/areas/ControlArea/SamplingArea/ResultsArea/SamplerOutputArea/HistogramsPanel.tsx

```diff
@@ -15,23 +16,36 @@ const HistogramsPanel: FunctionComponent<HistogramsProps> = ({
 }) => {
   const paramNamesResorted = useMemo(() => {
     // put the names that don't end with __ first
-    const names = paramNames.filter((name) => !name.endsWith("__"));
-    const namesWithSuffix = paramNames.filter((name) => name.endsWith("__"));
+    const names: [string, number][] = [];
+    const namesWithSuffix: [string, number][] = [];
+
+    for (const [index, name] of paramNames
+      .map(prettifyStanParamName)
+      .entries()) {
+      if (name.endsWith("__")) {
```

Comment:
Does this play okay with the prettifying? From how I read you here, you shouldn't have a case where a trailing __ gets hidden by bracketization since those are supposed to be numeric indices, but is that the case for tuples as well, those are always numeric?

And would we want to sort a dunder-suffixed tuple following this rule (e.g. `theta__.7 ` belongs with the regular parameters or not?) Or does that just not happen because the double-underscore-suffixed ones are system variables?

Reply:
It is illegal in Stan to end a variable with `__`, so the only ones that show up are the algorithmic ones, and those are always scalars

Reply:
Perfect. In that case I don't see any issues on visual inspection. What's a good model to check it out with?

Reply:
The Disease Transmission example has a bunch of array parameters. I don't think we have an example that uses tuples, but a simple one is something like

```stan
parameters {
  tuple(vector[10], real) foo;
}
model {
 foo.1 ~ normal(1,5);
 foo.2 ~ std_normal();
}
```